File: backup/basic_count.js
let in_count = 13;
let in_hold = 13;
let out_count = 13;
let out_hold = 13;

console.log("this works")
let run_count = function(in_count,in_hold,out_count,out_hold) {
    for(let i = 0; i < in_count; i ++) {
        print(i)
        
    }
} 

let testfunction = function() {
    print("works")
    // Example usage:
    pause(() =>  2);
}

function pause(callback, seconds) {
    setTimeout(callback, seconds * 1000);
}

File: backup/basic_count.py
# Basic thirteen count
import os

# [in_count,in_hold,out_count,out_hold]
count_array = [13,13,13,13]



os.system(f"say starting")
for i in count_array:
    print()
    print("level 1, i =",i)
    for j in range(i):
        number = j + 1
        os.system(f"say {number}")
        print("level 2, j =",j,"i =",i)